PyDMXControl/_Colors.py

```diff
@@ -62,11 +123,47 @@ def to_dict(colors: List[int]) -> Dict[str, int]:
 
     @staticmethod
     def to_tuples(colors: List[int]) -> List[Tuple[str, int]]:
-        """ Assumes RGBWA """
+        """Convert a color from a list to a list of tuples.
+        
+        Assumes RGBWA.
+        
+        Parameters
+        ----------
+        colors: Color to convert to tuples.
+
+        Returns
+        -------
+        List[Tuple[str, int]]
+            Color as a list of tuples.
+
+        Examples
+        --------
+
+        >>> PyDMXControl.Colors.to_tuples([1,2,3,4,5]) 
```

Comment:
```suggestion
        >>> Colors.to_tuples([1,2,3,4,5]) 
```
Remove package name.